lib/entry/StateManager.js

```diff
@@ -121,8 +121,9 @@ module.exports = function(liveEntry) {
                                 pushManager.setObjectEnabled(liveEntry.entryId, false);
                                 return backendClient.registerEntryInDatabase(liveEntry.entryObject, KalturaLiveStatus.PLAYABLE, event)
                                     .then((newEntryObj) => {
+                                        const firstTime = (lastServerUpdate == null);
                                         lastServerUpdate = new Date();
-                                        liveEntry.entryObjectUpdated(newEntryObj);
+                                        liveEntry.entryObjectUpdated(newEntryObj, firstTime);
```

Comment:
what about scenario of two start/stop preview at the same sessioN?  firstTime will still be false